local/mib2c-conf.d/generic-get-in_addr_t.m2i

```diff
@@ -0,0 +1,13 @@
+#############################################################  -*- c -*-
+## generic include for XXX. Do not use directly.
```

Comment:
Should "XXX" perhaps be changed into "in_addr_t"?

Reply:
sure..